src/metabase/cmd/copy.clj

```diff
@@ -346,7 +310,12 @@
 
 (def ^:private entities-without-autoinc-ids
   "Entities that do NOT use an auto incrementing ID column."
-  #{Setting Session DataMigrations ImplicitAction HTTPAction QueryAction})
+  #{:model/Setting
+    :model/Session
+    :model/DataMigrations
+    :model/ImplicitAction
+    :model/HTTPAction
+    :model/QueryAction})
```

Comment:
`:model/TablePrivileges` should also be in this list. Should there be a test that catches this?

Reply:
`metabase.cmd.load-and-dump-test/load-and-dump-test` failed because I forgot to add it there